app/server.js

```diff
@@ -41,7 +41,7 @@ server.use(function(req, res, next) {
 });
 
 // routes ======================================================================
-server.use(subdomain('api', require('./routes/api'))); // load our api routes and configure them to use the api subdomain
+server.use(subdomain('api', require('./routes/api')(passport))); // load our api routes and configure them to use the api subdomain
```

Comment:
because as far as I could tell, it needed to be the same instance which the server initializes. The alternative is having ALL the routes and passport config in server.js which is much messier. [I got this idea from here](https://scotch.io/tutorials/easy-node-authentication-setup-and-local)